prql-compiler/tests/integration-rdbms/docker-compose.yml

```
@@ -0,0 +1,47 @@
+services:
+  postgres:
+    image: "postgres:alpine"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: dummy
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: root
+    volumes: &vol
+      - ../integration/data/chinook:/tmp/chinook
+  mysql:
+    # No arm64 image available; remove when one does become available
+    platform: linux/amd64
+    image: "mysql:debian"
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_DATABASE: dummy
+      MYSQL_ROOT_PASSWORD: root
+    volumes:
+      - ./conf:/etc/mysql/conf.d
+      - ../integration/data/chinook:/tmp/chinook
```

> Wenn starting docker compose the owner of prql-compiler/tests/integration/data/chinook is changed on Linux.

I think it's because in Linux, the user in the container is root. In that case, bind mounting as read-only might solve the problem. (I have not tried if this works here!)

The ownership issue on bind mounting does not occur on macOS and Windows, which have different file systems. (Because host OS files are handled as root's)

```suggestion
    volumes: &vol
      - ../integration/data/chinook:/tmp/chinook:ro
  mysql:
    # No arm64 image available; remove when one does become available
    platform: linux/amd64
    image: "mysql:debian"
    ports:
      - "3306:3306"
    environment:
      MYSQL_DATABASE: dummy
      MYSQL_ROOT_PASSWORD: root
    volumes:
      - ./conf:/etc/mysql/conf.d:ro
      - ../integration/data/chinook:/tmp/chinook:ro
```

---

(I didn't want to add this without checking with @Jelenkee , but it does seem like a good idea)

---

Fixed in #2352